packages/flutter/test/painting/decoration_test.dart

```diff
@@ -326,7 +356,7 @@ void main() {
     expect(call.positionalArguments[3], isA<Paint>());
     final Paint paint = call.positionalArguments[3] as Paint;
     expect(paint.colorFilter, colorFilter);
-    expect(paint.color, const Color(0x7F000000)); // 0.5 opacity
+    expect(paint.color, _matchesColor(const Color(0x7F000000))); // 0.5 opacity
```

Comment:
Is this the only failure? if so, it looks like all its doing is testing for 0.5 opacity. You could query that directly

Reply:
It's the only one that showed up 🤞  The next PR is going to be the one that's probably going to break a lot of stuff when I update things like `lerp`.  We may need to put this color matcher somewhere else where it can be shared for that.

You mean something like `expect(paint.color.a, closeTo(0.5))`?  It seems likely that was the intent of the author.  I'd rather stick to what is actually asserted.

Reply:
Not sure what you mean?

Reply:
> it looks like all its doing is testing for 0.5 opacity.

I'm saying that may have been their intention but it is actually asserting a color so it's safer to just keep it asserting the same thing that is written.  If the goal is to avoid having to have a color matcher, we'll likely need one anyway to land https://github.com/flutter/engine/pull/54981

Reply:
kk